karma.shared.js

```diff
@@ -1,25 +1,19 @@
 'use strict';
 
 /* This is the shared karma config */
-
 var merge = require('lodash.merge');
-var RewirePlugin = require('rewire-webpack');
 var webpackConfig = require('./webpack.config');
 
-
 var newWebpackConfig = merge({}, webpackConfig);
 // Remove the bits from the shared config
 // that we don't want for tests.
 delete newWebpackConfig.output;
 delete newWebpackConfig.entry;
 
-// Add this just for testing:
-newWebpackConfig.plugins = [
-  new RewirePlugin(),
-];
+newWebpackConfig.plugins = [];
```

Comment:
I don't think it can because we don't want commons here. See https://github.com/mozilla/payments-ui/blob/master/webpack.config.js#L31